kv-components/vue/KvLoadingSpinner.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<svg
+		class="animate-spin"
+		:class="{
+			'h-3 w-3'
+				: size === 'small',
+			'h-4 w-4'
+				: size === 'medium',
+			'h-8 w-8'
+				: size === 'large',
+			'text-brand'
+				: color === 'brand',
+			'text-white'
+				: color === 'white',
+			'text-black'
+				: color === 'black',
+		}"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 24 24"
+	>
+		<g
+			clip-path="url(#clip0)"
+			stroke-width="2"
+		>
+			<circle
+				class="stroke-current text-gray-300 opacity-low"
+				cx="12"
+				cy="12"
+				r="8"
+				transform="rotate(135 12 12)"
```

Comment:
@eddieferrer did you get the rotation spec with the easing and timings from @itseunsoo or Nathan? I'm not sure a constant rotation is going to cover the design they specced out.

Reply:
I did not. I actually have not seen any animation from design. This is based on the Interim styleguide figma. Me, @ryan-ludwig, @itseunsoo had a short meeting to discuss the loader in general.